### DISCUSSION TITLE
[Bug] 无法通过docker环境变量设置默认助手为claude-3-5-sonnet

### DISCUSSION THREAD

**Discussion by hlc1209** [lobehub/lobehub#2996] [General Question] [answered]
### 📦 部署环境

Docker

### 📌 软件版本

1.1.3

### 💻 系统环境

Windows

### 🌐 浏览器

Chrome

### 🐛 问题描述

docker 环境变量配置为
DEFAULT_AGENT_CONFIG：model=claude-3-5-sonnet-20240620;chatConfig.autoCreateTopicThreshold=0;params.temperature=0.4
OPENAI_MODEL_LIST：-gpt-3.5-turbo,-gpt-4-turbo,+gpt-4,+gpt-4o

界面显示
![image](https://github.com/lobehub/lobe-chat/assets/45445824/73b481fc-8e58-49b0-b56c-f56429c1125f)

[a comment by an automated account is not shown]

**Comment by sxjeru** [comment, marked as answer]
可以试试这样写: `model=claude-3-5-sonnet-20240620;provider=anthropic`

[a comment by an automated account is not shown]
